modules/sdk/src/main/scala/org/tessellation/sdk/package.scala

```diff
@@ -47,4 +56,24 @@ package object sdk {
       EventTrigger.getClass -> 530,
       TimeTrigger.getClass -> 531
     ).union(sharedKryoRegistrar)
+
+  object PriorityPeerIds {
+    private val mainnet = Set(
```

Comment:
IMO it should be a non empty set here, and then passed to the Map with .some, as you expect it always to be present